book/website/communication/citable/citable-otherscite.md

```diff
@@ -0,0 +1,70 @@
+
+When citing research objects from other people/researchers, ensure you verify when, where, and how to properly cite those research objects.
+Below, we provide a brief explanation for each of these three scenarios.
+
+(cm-citable-otherscite-when)=
+## When to cite a research object?
+
+As a rule-of-thumb, every research object that plays a significant role in your research project should be cited in you publication or methodology paper. 
+For example, when using others software to analyse your research data, it is vital to cite and attribute it properly.
+In some cases, the contribution of a research object may be small but critical to your results.
+You should always consider about giving credit to others for their work, showing gratitude to them, and try to implement more repeatable science {cite:ps}`LaZerte2021`
+
+If you are unsure about what to cite because you doubt the contribution of a particular research object to your research, you can answer the following questions from {cite:ps}`ShouldICite`
+- Does the software ask you to cite it?
+- Did the software play a critical part in, or contributed something unique to, your research?
+- Did the software manipulate or create your data? This includes storage, visualization, and communication of your data and results.
+- Do the authors of the software rely on academic credit for funding? Look for academic institutions in email addresses, URLs.
+
+If the answer to any of these questions is yes, then you must/need to cite that research object.
+
+(cm-citable-otherscite-where)=
+## Where to cite a research object?
+
+You should directly cite research objects from others in you publication where relevant.
+For example, if specific software was used to process data, it should be cited in the section describing the processing. 
+Similarly, if all the figures in a publication are created using the same software, it may be more useful to mention that software in the main text or acknowledgements. 
+Another example is when videos or images from others are explicitly mentioned; this as an opportunity to cite those research objects. 
+In such cases, these research objects should commonly be included in the reference section. 
+
+However, it is important to note that citation practices may change depending on the publisher's guidelines.
+The best practice is to pair citations in the reference section with in-text citations {cite:ps}`Stall2023`.
+This approach avoids mentioning datasets or software in footnotes or only in supplementary materials, where clear citations may not be provided.
+In cases where a research object was used as a test or pilot but did not play a role in the final publication, it is possible to mention these in the acknowledgements section.
+This ensures proper credit is given, even if the object was not ultimately used.
+
+Finally, availability statements should usually be reserved for research objects produced during your project, and not for those created by others.
```

Comment:
```suggestion
Finally, availability statements should usually be reserved for research objects produced during your project, and not for those created by others.
```
```suggestion
```

If they used the data from others for their results they should provide that explanation in the data availability statement actually - so I would remove this.